speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
 AddonExecute execute addon {Addon}
 AddonExecute execute plugin {Addon}
 AddonExecute execute script {Addon}
+AddonExecute execute the addon {Addon}
+AddonExecute execute the plugin {Addon}
+AddonExecute execute the script {Addon}
+AddonExecute execute the {Addon}
```

Comment:
Yeah, I wouldn't worry too much more about it.  Adding the articles is obviously the right thing to do since we both were able to make it fail without them.